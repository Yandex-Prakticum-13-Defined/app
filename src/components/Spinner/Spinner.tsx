import React from 'react';
import './Spinner.scss';

const Spinner = () => (
  <div className='preloader'>
    <div className='preloader__row'>
      <div className='preloader__item' />
      <div className='preloader__item' />
    </div>
  </div>
);

export default Spinner;
