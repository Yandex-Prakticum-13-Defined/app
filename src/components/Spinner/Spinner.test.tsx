import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from './Spinner';

describe('Тестируем Spinner.tsx', () => {
  it('проверяем снимок', () => {
    const spinnerComponent = renderer.create(<Spinner/>).toJSON();
    expect(spinnerComponent).toMatchSnapshot();
  });
});
