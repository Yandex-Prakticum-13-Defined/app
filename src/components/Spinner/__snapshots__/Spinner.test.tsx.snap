// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Тестируем Spinner.tsx проверяем снимок 1`] = `
<div
  className="preloader"
>
  <div
    className="preloader__row"
  >
    <div
      className="preloader__item"
    />
    <div
      className="preloader__item"
    />
  </div>
</div>
`;
